django/contrib/admin/templates/admin/base.html

```diff
@@ -24,7 +24,7 @@
 
 <body class="{% if is_popup %}popup {% endif %}{% block bodyclass %}{% endblock %}"
   data-admin-utc-offset="{% now "Z" %}">
-
+  <a href="#content-start" class="skip-to-content-link">{% translate 'Skip to main content' %}</a>
```

Comment:
Here I inserted a `tab` because is inside of `body`, but then, `div#container` it's at the same level.

Let me know if the tabulation it's OK! I didn't want do a big diff.